codingStandards/CodingStandard-CSharp.md

```diff
@@ -0,0 +1,367 @@
+# C# Coding Standard
+
+**Important:** Use [CSharpGuidelines](https://github.com/dennisdoomen/CSharpGuidelines) for any topics not covered in this document.
```

Comment:
I feel that the CSharpGuidelines is more comprehensive for beginners. It includes basic stuff like casing convention, layout while the Microsoft one doesn't.